beetsplug/fetchart.py

```diff
@@ -1483,12 +1515,13 @@ def art_for_album(
                 # fulfill the requirements
                 for candidate in source.get(album, self, paths):
                     source.fetch_image(candidate, self)
-                    if candidate.validate(self):
-                        out = candidate
+                    validated_candidate = candidate.validate(self)
+                    if validated_candidate is not None:
+                        out = validated_candidate
                         self._log.debug(
-                            "using {0.LOC_STR} image {1}".format(
-                                source, util.displayable_path(out.path)
-                            )
+                            "using {0.LOC.value} image {1}",
```

Comment:
Personally, having to use `.value` to access the underlying value always makes me reconsider the need for the enum - unless it serves any other purpose except typing I end up defining it as a `Literal`.

Reply:
That's not necessary in principle — I did this to preserve the exact the exact debug output from before the change (lowercase instead of uppercase), but that is probably not worth it. I've just removed the `.value` now.